packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-range/gux-form-field-range.scss

```diff
@@ -35,7 +35,7 @@
 $range-active: var(--gse-ui-rangeSlider-bar-selected-backgroundColor);
 $track-width: 100%;
 $track-height: var(--gse-ui-rangeSlider-bar-height);
-$thumb-diameter: calc(var(--gse-ui-rangeSlider-handle-width) / 2);
+$thumb-diameter: calc(var(--gse-ui-rangeSlider-handle-height) / 2);
```

Comment:
Should this stay using the `width` token ?

Reply:
so I changed it because everything it's used in has to do with block margins and sizing, which is height in LTR from my understanding. It makes more sense to use height in this case imo.

Maybe we should talk to UX about the token naming too if we are switching everything to block and inline sizing?